sdl_android/src/main/java/com/smartdevicelink/util/BitConverter.java

```diff
@@ -1,10 +1,19 @@
 package com.smartdevicelink.util;
 
 public class BitConverter {
+	/**
+	 * @param bytes passing in a null byte array returns a null object

```

Comment:
The description of what's returned should be in the "return" tag Something like:
@param bytes byte array that will be converted to hex
@return the String containing converted hex values or null if byte array is null